### PR TITLE
Add go.mod update check for tools and web

### PR DESCRIPTION
**Pull request by shu-mutou**
- modules/common/certificates/go.mod
- modules/common/tools/go.mod
- modules/web/go.mod

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -10,6 +10,16 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 5
+  - package-ecosystem: "gomod"
+    directory: "modules/common/tools/"
```
Comment (maciaszczykm) [inline .github/dependabot.yml:14]: I'd change it to whole `modules/common` dir. We include K8s dependencies in `modules/api` anyway. In my opinion it makes sense as we will be notified about any releases.

Comment (shu-mutou) [reply .github/dependabot.yml:14]: Unfortunately dependabot does not check recursively. So if we want to check whole `module/common` directory, we need to add one more directive for `module/common/certificates`.

Comment (maciaszczykm) [reply .github/dependabot.yml:14]: Let's add it then.

Comment (shu-mutou) [reply .github/dependabot.yml:19]: Got it. Done.

**Pull request merged**